packages/components/src/styled-primitives/button/index.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
As we talked, this doesn't look useful outside of `/primitives/button/index.js`, we can move the file in the same folder, or even merge this into that file and remove this altogether.